prometheus-hanadb_exporter.spec

```diff
@@ -25,7 +25,7 @@
 %define _sysconfdir %{_prefix}/etc
 
 Name:           prometheus-hanadb_exporter
-Version:        0
+Version:        0.7.1
```

Comment:
If the version is to 0 I think we don't need to change , because the shap-bot Release agent will automatically submit the right version to pkg

Reply:
I was not aware of that automation, I will remove this change, thx.

Reply:
Yes, that's right, I forgot that.
This entry is created by the `_service` file, and the version is retrieved from github when the release is created (it can be done manually too, I guess)